README.md

```diff
@@ -38,21 +38,21 @@ TinyMCE is easily integrated into your projects with the help of components such
 - [tinymce-vue](https://github.com/tinymce/tinymce-vue)
 - [tinymce-angular](https://github.com/tinymce/tinymce-angular)
 
-With over 29 integrations, and 400+ APIs, see the TinyMCE docs for a full list of editor [integrations](https://www.tiny.cloud/docs/integrations/).
+With over 29 integrations, and 400+ APIs, see the TinyMCE docs for a full list of editor [integrations](https://www.tiny.cloud/docs/tinymce/6/integrations.html).
```

Comment:
This URL worries me, it's to a WIP page and I suspect it probably needs to link to the `How-to guides` -> `Environment setup guides` as we no longer have an integrations page and that is currently the nav that has the details about the integrations. I guess worse case scenario we can redirect this single URL later.

Reply:
Talked with Lee, we can deal with this with redirects if needed.